.circleci/config.yml

```diff
@@ -8,11 +8,10 @@ jobs:
         resource_class: large
         docker:
             -
-                image: circleci/golang:1.12-stretch
+                image: banzaicloud/golang:1.13
```

Comment:
Why not using the circleci image?

Reply:
We use mysqldiff, which is only available in stretch/bionic, but not in later distros.